README.md

```diff
@@ -3,3 +3,23 @@
 A small set of utility programs for OSv:
 List:
  - cd.so
+
+
+## template
+
+`template` program takes two paths as arguments. First path should point to
+the file that will serve as a template that gets expanded using environment
+variables similar to how `bash` expands variables in strings. Second path
+should point to the destination file. For example:
+
+    $ cat a.txt
+    Name of master is ${MASTER_NAME} and worker is ${WORKER_NAME}.
```

Comment:
Mention that $MASTER_NAME is not expanded, only ${MASTER_NAME}.

Reply:
I should probably specify template language. I will add a section about what walid templates are.